src/conn/conn_sweep.c

```diff
@@ -266,14 +266,13 @@ __sweep_server(void *arg)
     WT_DECL_RET;
     WT_SESSION_IMPL *session;
     uint64_t last, now;
-    uint64_t last_las_sweep_id, min_sleep, oldest_id, sweep_interval;
+    uint64_t min_sleep, sweep_interval;
```

Comment:
We don't need `min_sleep` anymore. We can just use `conn->sweep_interval` in the code.